corehq/apps/app_manager/app_strings.py

```diff
@@ -118,11 +118,6 @@ def maybe_add_index(text):
             yield _get_custom_icon_app_locale_and_value(custom_icon_form, custom_icon_text, module=module)
 
         if hasattr(module, 'report_configs'):
-            yield id_strings.report_menu(), 'Reports'
-            yield id_strings.report_name_header(), 'Report Name'
-            yield id_strings.report_description_header(), 'Report Description'
-            yield id_strings.report_last_sync(), 'Last Sync'
-            yield id_strings.report_data_table(), 'Data Table'
```

Comment:
Good eye. Yes, this will now be visible on all apps. This is a change, but it's consistent with how UI translations in general work; there's no infrastructure for including/excluding keys based on what functionality the app uses (though I'd like to add that eventually).

It's always been only included once per app, though. This code does run for every module, but since the key is the same for each module and ultimately we're building a dictionary, each key only appears once in the final file.

Reply:
Hmm.. so a user would translate something that wont be visible/useful for them. Sounds fine since its just a few translations for now.
> (though I'd like to add that eventually).

👍